src/components/appNavigation/actionBar/hooks/useActionLink.js

```diff
@@ -0,0 +1,72 @@
+import { useCallback, useState } from 'react'
+
+import { mapValues } from 'lodash'
+import {
+  fireEvent,
+  GOTO_DONATE,
+  GOTO_LEARN,
+  GOTO_MARKET,
+  GOTO_USEGD,
+  GOTO_VOTE,
+} from '../../../../lib/analytics/analytics'
+import { useUserStorage } from '../../../../lib/wallet/GoodWalletProvider'
+import Config from '../../../../config/config'
+import { openLink } from '../../../../lib/utils/linking'
+
+const { learnUrl, useGdUrl, donateUrl, voteUrl, marketUrl } = Config
+
+const addExtraProps = (value, key) => ({
+  ...value,
+  wasClickedProp: `${key}Clicked`,
+  icon: value.icon || key,
+})
+
+const externals = mapValues(
+  {
+    learn: {
+      url: learnUrl,
+      event: GOTO_LEARN,
+    },
+    usegd: {
+      url: useGdUrl,
+      event: GOTO_USEGD,
+    },
+    donate: {
+      url: donateUrl,
+      event: GOTO_DONATE,
+    },
+    vote: {
+      url: voteUrl,
+      event: GOTO_VOTE,
+    },
+    market: {
```

Comment:
do we still need good market btn ? 
if no - please remove this item and corresponding event constants from analytics

Reply:
removed